### ISSUE TITLE
[Bug]: HeyForm forms are wildly inaccessible

### ISSUE THREAD

**Issue by diondiondion** [heyform/heyform#244]
### Bug Description

HeyForm form inputs are not accessible to keyboard users and people using assistive technology like screen readers. For example, radio buttons are constructed entirely out of div elements and can not be tabbed to or activated using the keyboard. Field labels are marked up as headings and are not associated with their inputs. Input placeholders are used in place of labels. Focus indicators are missing or too subtle to be useful for navigation.

### Steps to Reproduce

Attempt to use HeyForms with a screen reader (e.g. VoiceOver), or use a keyboard and try tabbing through the controls to navigate.

### Expected Behavior

HeyForms should be accessible to all people, regardless of visual impairment or input method. Web standards and accessibility guidelines such as WCAG 2.1 should be followed as much as possible, to avoid exclusion of people and legal issues caused by non-compliance with the [EAA](https://commission.europa.eu/strategy-and-policy/policies/justice-and-fundamental-rights/disability/european-accessibility-act-eaa_en) in Europe or [ADA](https://www.ada.gov/resources/web-guidance/) in the US.

### HeyForm Installation

heyform.net

### Version

/

### Desktop OS

macOS

### Desktop Browser

Firefox

### Desktop Browser Version

148

### Mobile Device

None

### Mobile OS

None

### Mobile Browser

None

**Comment by dearroy** [comment]
Oh I didn't realize that.

Will take a look and try to fix it tomorrow.

**Comment by diondiondion** [comment]
@iMuFeng It doesn't look like this was actually addressed? I can't find any PRs or commits with accessibility fixes, and the templates on the HeyForm website are still full of obvious accessibility failures.
